src/clean_env_before_mining.py

```diff
@@ -6,10 +6,12 @@
 The script can be run from the command line:
     * python3 clean_env_before_mining.py
 """
-
+import logging
 import os
 import shutil
 
```

Comment:
```suggestion
import shutil

# Configure logging
logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(levelname)s - %(message)s')


```

Reply:
Logging is configured only in first script. I am not sure, if this is the best practice, articles differ with this topic. Will dive into this problematic later in refactoring issue focused on logging. Change can be seen in commit: [fb78190](https://github.com/AbsaOSS/living-doc-generator/pull/16/commits/fb78190e5adedc65ded39abff55adc6f51da5407).